From 67768e44745cbf92a84105f2db3cec33f05d143d Mon Sep 17 00:00:00 2001
From: Sudaku <lollipop.studio.cn@gmail.com>
Date: Sat, 2 Jun 2018 14:33:49 +0800
Subject: [PATCH 20/29] arm64: dts: rockchip: enable display nodes on
 rk3328-roc-cc

Enable the necessary nodes to get output on the hdmi port of the board
---
 arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
index f666942f9..d76ea92f2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
@@ -113,6 +113,14 @@
 	status = "okay";
 };
 
+&hdmi {
+	status = "okay";
+};
+
+&hdmiphy {
+	status = "okay";
+};
+
 &i2c1 {
 	status = "okay";
 
@@ -296,3 +304,11 @@
 &usb_host0_ohci {
 	status = "okay";
 };
+
+&vop {
+	status = "okay";
+};
+
+&vop_mmu {
+	status = "okay";
+};
-- 
2.21.0

